citations-tool/tool/src/webapp/js/new_resource.js

```diff
@@ -560,7 +560,7 @@ citations_new_resource.init = function() {
 		return false;
 	});
 	$('#PickResource').on('click', function(eventObject) {
-		var target = $(eventObject.target);
+		/*var target = $(eventObject.target);
```

Comment:
You shouldn't leave commented out code in unless there's a really good reason, if someone want it back they can look in the git history. If there is a good reason document it with a comment about the commented out section.